ts/Data/DataOnDemand.ts

```diff
@@ -85,6 +89,37 @@ export class DataOnDemand {
      *
      * */
 
+    /**
+     * Loads the store of the source.
+     *
+     * @function Highcharts.DataOnDemand#getSource
+     *
+     * @param {string} name
+     * Name of the source.
+     *
+     * @return {Promise<Highcharts.DataStore>}
+     * Returns the store of the source.
+     */
+    public getSource(
+        name: string
+    ): Promise<DataStore<DataStore.Event>> {
+        const sources = this.sources;
```

Comment:
Do we go with 
```
const oneTing;
const secondThing;
```
or
```
const oneTing,
    secondThing;
```
?

Reply:
I probably mixed up personal styling and Highsoft styling.